src/components/DateInput.js

```diff
@@ -270,7 +269,7 @@ export default class DateInput extends React.Component {
       dateFormat, defaultValue, keyboard, locale, onBlur, onChange, parse, positionFixed, value, state, ...props } = this.props; // eslint-disable-line no-shadow
     const { open } = this.state;
     const date = this.getCurrentDate();
-    const dropdownProps = open ? { positionFixed } : {};
+    const dropdownProps = open ? { strategy: positionFixed ? 'fixed' : undefined } : {};
```

Comment:
How about this instead?
```suggestion
    const dropdownProps = open && positionFixed ? { strategy: 'fixed' } : {};
```

Reply:
Yup, that looks better as well!

Reply:
Updated